### PR TITLE
fix: Make sure unsigned bootstrap artifacts can always be installed.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: None
Ticket: MEN-6934

Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: src/artifact/parser.cpp [hunk 1]
```diff
@@ -138,7 +139,8 @@ ExpectedArtifact Parse(io::Reader &reader, config::ParserConfig config) {
 		tok = lexer.Next();
 
 		// Verify the signature
-		if (config.artifact_verify_keys.size() > 0) {
+		if (config.verify_signature != config::Signature::Skip
```
[the remark this answers is not in the document]
Comment (kacf) [reply src/artifact/parser.cpp:142]: Oh, you're right, I should pass it in instead. I missed that parameter.

[the remark this answers is not in the document]
Comment (kacf) [reply src/artifact/parser.cpp:142]: But yeah, maybe this is clearer. The alternative won't work for unsigned artifacts, so I think I'll keep my approach and just remove the argument altogether.

[the remark this answers is not in the document]
Comment (kacf) [reply src/artifact/parser.cpp:142]: Done.

**Review by lluiscampos — APPROVED** [review]
Looks good to me!

**Review by oleorhagen — APPROVED** [review]

**Pull request merged**